sdk/nodejs/runtime/settings.ts

```diff
@@ -533,3 +536,10 @@ export async function monitorSupportsDeletedWith(): Promise<boolean> {
 export async function monitorSupportsAliasSpecs(): Promise<boolean> {
     return monitorSupportsFeature("aliasSpecs");
 }
+
+/**
+ * Returns true if we throw exceptions when attempting to convert outputs to strings.
+ */
+export function throwOutputToString(): boolean {
+    return options().throwOutputToString ?? false;
```

Comment:
Given this is just an envvar it should probably be in sdk/nodejs/utils.ts, not here and not on the settings object. See how disableResourceReferences is written.

Reply:
Interesting, I was basing this off of `cacheDynamicProviders` / `PULUMI_NODEJS_CACHE_DYNAMIC_PROVIDERS` which is also only an env var. Seems simpler to do int the `disableResourceReferences`, but on the other hand that is adding to a generic `utils.ts` module instead of a more specific `settings.ts` module.

Reply:
`cacheDynamicProviders` is being needlessly complex in this case, do the same as `disableResourceReferences`. I'll change `cacheDynamicProviders` to this style as well when I've got 10 minutes.